localstack/aws/accounts.py

```diff
@@ -103,9 +103,10 @@ def get_account_id_from_access_key_id(access_key_id: str) -> str:
         if not config.PARITY_AWS_ACCESS_KEY_ID:
             # If AWS_ACCESS_KEY_ID has production AWS credentials, ignore them
             if access_key_id.startswith("ASIA") or access_key_id.startswith("AKIA"):
-                LOG.warning(
```

Comment:
I would prefer changing the log level here to `debug` than adding yet another config option which simply supresses a single log message.

Reply:
Also ok for me

Reply:
Hard to say, I think. People are using DEBUG=1 very often, so it might still spam them. Then, on the other hand, I think warning is the correct log level, as it is something unexpected happening, not just some site note to a expected flow.

I am okay with either way. I agree with this approach not scaling in the future but on the other hand it does not really fit the "debug" logs.

In any case, I think it only solves #8380, and not the other issue (we need to investigate where these "production" credentials come from there).